### DISCUSSION TITLE
Why is only the readme file being published to github pages

### DISCUSSION THREAD

**Discussion by michaelstanwyck** [community/community#22813] [Other Feature Feedback, Questions, & Ideas]
<p>I saw another topic like this but it didn’t help me understand what I am doing wrong here.</p>
<p>I have a repo here: <a href="https://github.com/michaelstanwyck/zenclubla" rel="nofollow noopener">https://github.com/michaelstanwyck/zenclubla</a></p>
<p>When I go here: <a href="https://michaelstanwyck.github.io/zenclubla/" rel="nofollow noopener">https://michaelstanwyck.github.io/zenclubla/</a></p>
<p>I get a 404 error</p>
<p>Previously I was only getting the readme page so I deleted the repo and started over. Now I can’t figure out what the heck I’ve done.</p>
<p>Can someone point me towards what I need to do to get my site to display? Thanks</p>

**Comment by GASOLINE** [comment, marked as answer]
<p>Maybe if you rename zen-index.html to index.html it works?</p>
<p>I can see your site at: <a href="https://michaelstanwyck.github.io/zenclubla/zen-index.html" rel="nofollow noopener">https://michaelstanwyck.github.io/zenclubla/zen-index.html</a></p>
<p>More info (you need to publish first): <a href="https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages#default-source-settings-for-repositories-without-the-username-naming-scheme" rel="nofollow noopener">https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages#default-source-settings-for-repositories-without-the-username-naming-scheme</a></p>
<p>Example: <a href="https://gasoline.github.io/amber/" rel="nofollow noopener">https://gasoline.github.io/amber/</a></p>
<p>P.S. <em>It takes a few minutes before your changes are visible, so you can’t see your changes right away.</em></p>

**Comment by michaelstanwyck** [comment]
<p>Thank you! That is interesting. I will see if that helps but I am glad to see it published live!</p>

**Comment by concodex93** [comment]
<p>Hey </p>
<p>I’m getting the same issue where I can only see my README.md file <a href="https://concodex93.github.io/" rel="nofollow noopener">https://concodex93.github.io/</a>. What was the fix in the end? </p>
<p>Cheers,</p>
<p>Conor</p>

**Comment by daryllman** [comment]
<p>A little late, but this may be a fix to your problem:</p>
<p><a href="https://medium.com/@daryllwong/github-pages-does-not-work-for-me-f9d48fd44a5f" rel="nofollow noopener">https://medium.com/@daryllwong/github-pages-does-not-work-for-me-f9d48fd44a5f</a></p>

**Comment by cheryltky** [comment]
<p>I am also getting the same issue with the readme.md file being published instead of (about.Rmd and index.Rmd). Still couldn’t find an answer to the fix even after deleting the repo and restarting as well as following the links to articles provided in this thread. Could anyone help me figure it out?<br>
my github page: <a href="https://cheryltky.github.io/" rel="nofollow noopener">https://cheryltky.github.io/</a><br>
my repo : <a href="https://github.com/cheryltky/cheryltky.github.io.git" rel="nofollow noopener">https://github.com/cheryltky/cheryltky.github.io.git</a></p>

**Comment by vikrant-3009** [comment]
<p>Hey, this solution helped me. Thanks.<br>
I changed the name of my html file from index1 to index.html &amp; it worked.</p>
